internal/xds/rbac/converter.go

```diff
@@ -72,10 +75,24 @@ func getCustomConfig(config *anypb.Any) (json.RawMessage, string, error) {
 			return nil, "", fmt.Errorf("failed to unmarshal resource: %v", err)
 		}
 		return convertCustomConfig(typedStruct.TypeUrl, typedStruct.Value)
+	case stdoutType:
+		stdoutLoggerConfig := &v3auditloggersstreampb.StdoutAuditLog{}
+		if err := config.UnmarshalTo(stdoutLoggerConfig); err != nil {
+			return nil, "", fmt.Errorf("failed to unmarshal resource: %v", err)
+		}
+		return convertStdoutConfig(stdoutLoggerConfig)
 	}
 	return nil, "", fmt.Errorf("custom config not implemented for type [%v]", config.GetTypeUrl())
 }
 
+func convertStdoutConfig(config *v3auditloggersstreampb.StdoutAuditLog) (json.RawMessage, string, error) {
+	json, err := json.Marshal(config)
```

Comment:
Instead of calling this, you could simply return `{}`.

Reply:
Discussed offline, we don't want to simply ignore the input config and return `{}`, though it would be fine with the current impl